pyscriptjs/src/remote_interpreter.ts

```diff
@@ -105,10 +114,18 @@ export class PyodideInterpreter extends Interpreter {
     }
     /* eslint-enable */
 
+    /**
+     * delegates the setting of JS objects to
+     * the underlying interface.
+     * */
```

Comment:
this comment seems wrong. It's not "settings JS objects", it's registering JS modules.
https://pyodide.org/en/stable/usage/api/js-api.html#pyodide.registerJsModule

Reply:
thanks, changed!